platform-cloud/docs/orgs-and-teams/organizations.mdx

```diff
@@ -18,15 +18,32 @@ You can also add external collaborators to an organization.
 3. Enter any other optional fields as needed: **Description**, **Location**, **Website URL**, and **Logo**.
 4. Select **Add**.
 
-### Edit an organization
+### Organization settings 
+
+Owners can view and manage organizations from the **Settings** tab. The card at the top of the page displays organization metadata (Name and Full name, Website URL, Location, and Description).
+
+#### Credits 
+
+[Seqera Compute](../compute-envs/seqera-compute.mdx) environments consume credits when running pipelines or studio sessions. Credits are consumed for CPU time, memory and storage usage, and network costs. 1 Seqera Compute credit is equivalent to $1 (USD), and resources are charged for at the following rates:
+- CPU time: 1 CPU/Hr = 0.1 credit
+- Memory: 1 GiB/Hr = 0.025 credit 
+- Storage: 1 GB = 0.25 credit per month 
+
+:::note 
+Storage and network costs vary per region, charged at standard AWS rates. Data ingress and egress across regions incure additional cost. 
+:::
+
+Your available credit balance depends on the credits purchased and limits applied to your Seqera license. The **Credits** view contains the current credit balance available to the organization, and the total credits spent in the organization's workspaces. Select **Contact us to upgrade** to request additional credits for your organization. 
+
+#### Edit or delete organization 
 
 :::note
 From version 23.2, **organization owners** can edit their organization name, either from the organizations page or the [Admin panel](../administration/overview.mdx).
 :::
 
 As an **organization owner**, access the organization page from the organizations and workspaces dropdown, or open the user menu and select **Your organizations** to view and edit your organizations. As a root user, you can also edit organizations from the [Admin panel](../administration/overview.mdx).
 
-Open the **Settings** tab on the organization page, and select **Edit** in the **Edit Organization** row. Update the settings and select **Update** to save.
+Select **Edit** in the **Edit Organization** row to update the organization name, full name, description, location, website URL, and logo. Update the settings and select **Update** to save.
```

Comment:
This capitalization is different to the previous instance of this content. Let's align it with what's in the UI even if it's not aligned with our style guide.